lib/src/main/kotlin/at/bitfire/ical4android/AndroidCompatTimeZoneRegistry.kt

```diff
@@ -67,13 +69,13 @@ class AndroidCompatTimeZoneRegistry(
            but most Android devices don't now Europe/Kyiv yet.
            */
         if (tz.id != androidTzId) {
-            logger.warning("Using Android TZID $androidTzId instead of ical4j ${tz.id}")
+            logger.info("Using ical4j timezone ${tz.id} data to construct Android timezone $androidTzId")
 
             // create a copy of the VTIMEZONE so that we don't modify the original registry values (which are not immutable)
             val vTimeZone = tz.vTimeZone
             val newVTimeZoneProperties = PropertyList(vTimeZone.properties)
-            newVTimeZoneProperties.removeAll { property ->
-                property is TzId
+            newVTimeZoneProperties.removeAll { prop ->
+                prop is TzId || prop is TzUrl || prop.name == "X-LIC-LOCATION"
```

Comment:
Can we take a positive approach instead? So that we take only the elements we need (the TZID and the actual definitions) and drop the rest. We never know what the next ical4j version adds…

Reply:
Then I'd say that we ignore all properties and just add the `TZID`, since `STANDARD` and `DAYLIGHT` are components, not properties, and I think we don't need any other properties.